src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -172,6 +177,29 @@ registerVisualizationLayerDefinition({
   },
 });
 
+registerVisualizationLayerDefinition({
+  identifier: "fontra.glyph.locking",
+  name: "Glyph locking",
+  selectionMode: "editing",
+  userSwitchable: true,
+  zIndex: 700,
+  screenParameters: {
+    iconSize: 15,
```

Comment:
I tried with 12, and for me that's large enough.

Reply:
done